tests/js/realm-tests.js

```diff
@@ -348,26 +351,26 @@ module.exports = BaseTest.extend({
         new Realm({schema: [NotIndexed], path: '1'});
 
         TestCase.assertThrows(function() {
-            IndexedTypes.properties = { floatCol: {type: 'float', indexed: true} }
-            new Realm({schema: [IndexedTypes], path: '2'});
+            schemas.IndexedTypes.properties = { floatCol: {type: 'float', indexed: true} }
```

Comment:
I'm concerned by modifying a shared object like this since it might be shared with other tests. Let's at least create a shallow copy of `schemas.IndexedTypes` at the beginning of this test and use that instead?